discovery_engine_core/web-api/src/main.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+//! Web service that uses Xayn Discovery Engine.
+
+#![forbid(unsafe_code, unsafe_op_in_unsafe_fn)]
+#![deny(
+    clippy::future_not_send,
+    clippy::pedantic,
+    noop_method_call,
+    rust_2018_idioms,
+    unused_qualifications
+)]
+#![warn(unreachable_pub, rustdoc::missing_crate_level_docs)]
+#![allow(
+    clippy::items_after_statements,
+    clippy::missing_errors_doc,
+    clippy::module_name_repetitions,
+    clippy::must_use_candidate
+)]
+
+use std::{convert::Infallible, env, net::IpAddr};
+use uuid::Uuid;
+use warp::{hyper::StatusCode, Filter};
+
+#[tokio::main]
+async fn main() {
+    // TODO: TY-3011 - add filepath env var for documents data json file
+    let port = env::var("PORT")
```

Comment:
❓ can we be sure that there are no name conflicts? this env var name is quite general, maybe we should prefix it like `DE_PORT` or sth else. similarly for the `IP_ADDR`.